mqtt/mqtt-bridge/src/pump/mod.rs

```diff
@@ -16,7 +16,7 @@ use futures_util::{
     join, pin_mut,
 };
 use mockall::automock;
-use parking_lot::Mutex;
+use parking_lot::RwLock;
```

Comment:
just curious why this change in this PR and not separate?

Reply:
I started to use more read operations with changes in this PR: reading when retry timeout expired. so they can come together